### PR TITLE
[DOCS-1533] - Convert the README from markdown to Asciidoc

### PR DESCRIPTION
**Pull request by ddl-lgaddie**
### Link to JIRA

[DOCS-1533](https://dominodatalab.atlassian.net/browse/DOCS-1533)

### What issue does this pull request solve?

Converting the README to Asciidoc enables the docs team to dynamically include this file on the docs site.  That way, we can update the python-domino docs in one place instead of two.

Starting with 5.3.0, the README content will replace this out-of-date page:
https://docs.dominodatalab.com/en/5.2/api_guide/d9cc25/python-wrapper-for-domino-api/

Assuming this works well for everyone, we can do the same on any relevant release branches and dynamically include earlier versions of the README in earlier versions of the guide above, eventually eliminating the redundant and out-of-date file currently published there.

To do: Copyediting and probably some conditional directives to distinguish GitHub links from docs site links.

### What is the solution?

1. Convert the README from markdown to Asciidoc for compatibility with the docs site.
2. In the docs repo, create a new page and use the Asciidoc `include` directive to point to the remote file.  

The page is auto-updated every time the site rebuilds.

### Testing

GitHub preview:
https://github.com/dominodatalab/python-domino/blob/DOCS-1533-convert-readme-to-asciidoc/README.adoc

Docs site preview:
https://github.com/cerebrotech/documentation/pull/782/checks
Click *Gatsby Cloud* > *Gatsby Build Service*, then click the link and go to *API Docs* > *The python-domino Library*.

- [ ] Unit test(s)

### Pull Request Reminders

- [ ] Has relevant documentation been updated?
- [ ] Does the code follow [Python Style Guide] (https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html)
- [ ] Update the [changelog](https://github.com/dominodatalab/python-domino/blob/master/CHANGELOG.md)
- [x] Are the existing unit tests still passing?
- [ ] Have new unit tests been added to cover any changes to the code?
- [x] Has the JIRA ticket(s) been linked above?

### CODE REVIEW

**Comment by ddl-lgaddie** [comment]
See this related PR: https://github.com/cerebrotech/documentation/pull/782

. Go to **Checks** > **Gatsby Build Service**.
. Click the link to the preview build.
. Go to **API Docs** > **The python-domino Library**.

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,389 @@
+= python-domino
+
+This library provides bindings for the https://dominodatalab.github.io/api-docs/[Domino Data Lab API].
```
Comment (ddl-olsonJD) [inline README.adoc:3]: let's also mention that this is a legacy document, and may not reflect the most up today api,
or update link to current public api docs?

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+Add collaborators to a project.
+
+[source,python]
+----
+domino.collaborators_add("collaborator1", "collaborator2")
+----
+
+.Parameters
+
+* _collaborator_username:_ Usernames of one or more collaborators to add to the project.
```
Comment (ddl-lgaddie) [inline README.adoc:114]: @ddl-olsonJD I noticed these in the example code file and added them here.  Let me know if that's not correct.

Comment (ddl-lgaddie) [reply README.adoc:114]: I also tried a different presentation, here and below, to make the headings shorter while still including example syntax below each heading.  Let me know whether that works or whether it's misleading to our audience.  If it works, I'll do the same with the rest of the headings, and if not, I'll change this to match how the existing headings are formatted.

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+Remove tags from a project.
+
+[source,python]
+----
+domino.tags_remove("old")
+----
+
+.Parameters
+
+* _tag_name:_ Names of one or more tags to remove from the project.
```
Comment (ddl-lgaddie) [inline README.adoc:144]: @ddl-olsonJD Here are the new methods for tags.  Let me know whether this is correct.

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+
+List the IDs of a filtered datasets for a particular project.
+// What does this mean?
+
+* _project_id:_ The project identifier.
+
+==== datasets_names(project_id)
+
+List the names of a filtered datasets for a particular project.
+// What does this mean?
```
Comment (ddl-olsonJD) [inline README.adoc:366]: dido

File: README.adoc [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+* _dataset_description:_ (Optional) New description of the dataset.
+
+==== datasets_remove(dataset_ids)
+
+Delete a set of datasets.
+
+* _dataset_ids: (list[string])_ List of IDs of the datasets to delete.
+NOTE: Datasets are first marked for deletion, then deleted after a period of time.
+
+// How much time?
```
Comment (ddl-olsonJD) [inline README.adoc:400]: about 5 mins but may vary

File: README.adoc [hunk 6]
```diff
@@ -0,0 +1,453 @@
...
+
+* _project_name:_ The name of the project.
+* _owner_username:_ (Optional) The owner username for the project. 
+This parameter is useful when you need to create a project under an organization.
+
+==== collaborators_get()
+
+Get the list of collaborators on a project.
+
+==== collaborators_add()
```
[the remark this answers is not in the document]
Comment (ddl-lgaddie) [reply README.adoc:103]: > collaborators_add(username_or_email, message=""):

@ddl-olsonJD What does the `message` param do?

File: README.adoc [hunk 7]
```diff
@@ -0,0 +1,453 @@
...
+* _unpublishRunningApps:_ (Defaults to True) Check for active app instances in the current project and unpublish them before publishing.
+* _hardwareTierId:_ (Optional) Launch the app on the specified hardware tier.
+
+==== app_unpublish()
+
+Stop all running apps in the Domino project.
+
+=== Jobs
+
+==== job_start(_command*, _commit_id=None_, _hardware_tier_name=None_, _environment_id=None_, _on_demand_spark_cluster_properties=None_):
```
[the remark this answers is not in the document]
Comment (ddl-lgaddie) [reply README.adoc:261]: @ddl-olsonJD Would you please make that change on this branch to make sure it's done correctly?

File: README.adoc [hunk 8]
```diff
@@ -0,0 +1,446 @@
...
+
+[cols=",^"]
+|===
+| Domino Versions | Python-Domino
+
+| 3.6.x or lower
+| http://github.com/dominodatalab/python-domino/archive/0.3.5.zip[0.3.5]
+
+| 4.1.0 or higher
+| https://github.com/dominodatalab/python-domino/archive/1.0.0.zip[1.0.0] or higher
```
Comment (ddl-lgaddie) [inline README.adoc:22]: @ddl-olsonJD Is it possible to provide more granularity here?  Would it be useful to provide links to the full set of`.zip` files?

**Comment by ddl-lgaddie** [comment]
Since Asciidoc is not supported by the python package manager, this README should remain in Markdown.
The docs team will create an Asciidoc copy of the README for the docs site.  I'll open a new PR with the Markdown version of the updated README from this branch.

**Pull request closed without merging**